js/constants/countries.js

```diff
@@ -0,0 +1,1000 @@
+const countries = [
```

Comment:
This list doesn't need to have the names anymore, since those are localized in the `countries.properties` file.  Do you think we can rename this as `countryCodes.js` and then only store the country codes themselves?